.travis.yml

```diff
@@ -7,15 +7,33 @@ cache:
   directories:
     - lektor/admin/node_modules
 
+addons:
+  firefox: "47.0.1"
+
 before_install:
   - curl -s -o $HOME/.nvm/nvm.sh https://raw.githubusercontent.com/creationix/nvm/v0.31.4/nvm.sh
   - source $HOME/.nvm/nvm.sh
   - nvm install stable
   - node --version
+  - pip install -U pip 
+  - pip install pytest==2.9.2
+  - pip install pytest-mock
```

Comment:
If we need `pytest-mock`, then it should be in the `tests_require` list in the `setup.py` file.